ArcGISRuntimeSDKQt_CppSamples/Layers/BrowseOGCAPIFeatureService/BrowseOGCAPIFeatureService.cpp

```diff
@@ -206,10 +216,23 @@ void BrowseOGCAPIFeatureService::clearExistingFeatureLayer()
 
 void BrowseOGCAPIFeatureService::addFeatureLayerToMap()
 {
+  setLoading(false);
```

Comment:
It seems like there should be a simpler way where you don't need to be managing state throughout the code and setting/unsetting the loading state. Under which conditions would 'isLoading' be true? Is it just `if (m_featureService && m_featureService->loadStatus() == LoadStatus::Loading)`? Ideally we can let the API tell us when it is loading or not, but not sure if you have additional app level loading required here...

Reply:
I agree with @ldanzinger. `setLoading` seems unnecessary when the API itself can tell you this information.